package.json

```diff
@@ -3,9 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "yarn env && vue-cli-service serve",
```

Comment:
This may overwrite the existing env, and `yarn env` should only be run once. Could we keep it as it is?